CMSgov/onemac
# Oy2 19950

Story: https://qmacbis.atlassian.net/browse/OY2-19950
Endpoint: See github bot comment

### Details

Add Waiver RAI submission form

### Changes

- added seed data for a waiver in RAI Issued status
- added lambda handler for submitWaiverRai
- added waiver rai form and config
- connected routes to new form
- connected rai action to new form

### Implementation Notes

- Since only initial waivers are completed (no renewal or amendment yet) that is the only rai waiver tested and hooked up in the rai action object

### Test Plan

1. Login as state submitter
2. Go to package view waivers tab and locate a waiver in status of RAI Issued
3. Click Kebab and select RAI Response OR go to waiver detail page and select from the Action list
4. Verify you are redirected to an empty Waiver Rai Response with appropriate form title and sections as described in the ACs
5. Verify the parent waiver number appears in the "Waiver Number" as an uneditable text element.
6. Verify the required documents per the ACs
7. Submit
8. Verify you are redirected to the package dashboard and displayed the green success message
9. Verify an email is sent to the package triage team
